_posts/2018-11-16-grpc-dotnet-build.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: gRPC Meets .NET SDK And Visual Studio
```

Comment:
I like the title, but would be great to add a subtitle that would explain what readers can gain from this article (basically it's protobuf codegen integration for C#)

Reply:
*gRPC Meets .NET SDK And Visual Studio: Automatic Codegen On Build* now -- do you think this is ok? Not *too* catchy?

Reply:
I think that's a good title for a blogpost.